### DISCUSSION TITLE
how to include javascript file from node_modules?

### DISCUSSION THREAD

**Discussion by tombohub** [remix-run/remix#3706] [Q&A] [answered]
I have installed bootstrap library and included bootstrap.css via links().

But how do you include bootstrap.js? It's in node_modules.

**Comment by kiliman** [comment, marked as answer]
You might want to consider `react-bootstrap` so you can import normally.

https://react-bootstrap.github.io/getting-started/introduction

**Reply by tombohub** [reply]
Thanks, yes I've tried , but prefer regular bootstrap. Just a general way, is it possible to include javascript file?

**Reply by kiliman** [reply]
Have you tried: `import bootstrap from 'bootstrap'`

https://getbootstrap.com/docs/5.2/getting-started/download/#npm

**Reply by tombohub** [reply]
yes, it doesn't work.

**Reply by kiliman** [reply]
If you just want to use bootstrap, then I would simply add a `<script/>` tag like normal HTML.
```html
<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/js/bootstrap.min.js" crossorigin="anonymous"></script>
```

**Reply by tombohub** [reply]
Ok, thank you
